spec/decorators/member_decorator_spec.rb

```diff
@@ -1,4 +1,30 @@
+# frozen_string_literal: true
+
 require 'rails_helper'
+require 'draper/test/rspec_integration'
 
 RSpec.describe MemberDecorator do
+  describe '#hours_worked_in_the_three_last_months' do
+    let(:generate_member_decorator) { described_class.new(build_stubbed(:member)) }
+
+    context 'when active admin view is generated' do
+      it 'return the html version' do
+        member_decorator = generate_member_decorator
+
+        content = member_decorator.hours_worked_in_the_three_last_months
+
+        expect(content).to include('<p>')
+      end
+    end
+
+    context 'when the csv is generated' do
```

Comment:
Same as above, `'when the :csv option is true'`

Reply:
done